apps/participants/views.py

```diff
@@ -206,6 +207,7 @@ def get_teams_and_corresponding_challenges_for_a_participant(request, challenge_
     serializer = ChallengeParticipantTeamListSerializer(ChallengeParticipantTeamList(challenge_participated_teams))
     response_data = serializer.data
     response_data['is_challenge_host'] = is_challenge_host
+    response_data['time'] = timezone.now().replace(microsecond=0).strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
@Ram81 sorry to say but this is not the right way of adding data to the response. You have to add this fied in the serializer and write test for that.

Reply:
@deshraj I will do the required changes :+1:

Reply:
@deshraj but time is a field that we are calculating each time api endpoint gets called so do we still need to add it through a serializer?.